posts/2022-06-06-introduction-cypress-en.adoc

```diff
@@ -0,0 +1,216 @@
+= Introducing Cypress
+VincentBrule
+v1.0, 2022-06-06
+:title: Introducing Cypress
+:imagesdir: ../media/2022-06-06-introduction-cypress
+:lang: en
+:tags: [test, cypress, web, js, ts]
+
+This article is the beginning of a series of three articles dedicated to https://www.cypress.io[Cypress]:
+
+. Introducing Cypress
+. Going Further with Cypress
+. Comparing Cypress and its Competitors
+
+The goal of this first article is to make you want to discover and use *Cypress*. 
+To do so, we will present this tool and put it into practice with a simple example.
+
+== I/ What is Cypress used for?
+
+Tests are and always have been an important point in a project. 
+There are many types of testing such as unit testing and integration testing. 
+Cypress focuses on end-to-end testing (E2E).
+The E2E tests allow us to validate the user experience through predefined scenarios. 
+These scenarios will describe a group of actions that an average user will perform on our site. 
+This kind of test does not care about the internal implementation of our product, but will adopt the point of view of a user to verify the general behavior of our product.
```

Comment:
```ssuggestion
These tests do not care about the internal implementation of our product, but will adopt the point of view of a user to verify the general behaviour of our product.
```
It's perhaps a little pedantry, but in the previous sentences you refer to test (plural), then suddenly you switch to the singular. No biggy if you leave it, just change the typo "behavior" to "behaviour"

Reply:
That's a good point, thanks 👍